### ISSUE TITLE
sGrid missing?

### ISSUE THREAD

**Issue by taugustyn** [juliancwirko/react-boilerplate#12]
This example was meant to use sGrid but uses Sass instead? Or am I missing something?

**Comment by juliancwirko** [comment]
Yes :/ I'm sorry, I need to update the sGrid website. But it should be simple to use it with sGrid. You just need to remove scss, add stylus and import sgrid from npm. You can also find the sGrid implementation in git history : https://github.com/juliancwirko/react-boilerplate/commits/master
